crates/router/src/core/payments/operations/payment_start.rs

```diff
@@ -248,7 +248,8 @@ where
         BoxedOperation<'a, F, api::PaymentsStartRequest>,
         Option<api::PaymentMethodData>,
     )> {
-        helpers::make_pm_data(Box::new(self), state, payment_data).await
+        Ok((Box::new(self), None))
+        // helpers::make_pm_data(Box::new(self), state, payment_data).await
```

Comment:
You can remove this commented line of code.

Reply:
I have made the required changes. Please have a look once.